Magento_Theme/templates/custom_footer.phtml

```diff
@@ -6,6 +6,5 @@
 ?>
 
 <div>
-    <h2><?= $escaper->escapeHtml($block->getData('custom_footer_text')); ?></h2>
-    <p> <?= $escaper->escapeHtml($block->getData('new_custom_footer_text')); ?> </p>
+   <p class="current-date-text">Today's date: <?= $escaper->escapeHtml($block->getCurrentDate()) ?></p>
```

Comment:
all the phtml templates have to be cached. restore the `full_page` cache and you will see that the time isn't updated
```
bin/magento cache:enable full_page block_html
```
refresh the page in the browser a few times.
try not to use BE stuff for now, not to spend time on resolving unnecessary issues and it'd be better to focus on FE instead

Reply:
BTW, don't forget to wrap text to the translation function.
```html
<p class="current-date-text">
<?= $escaper->escapeHtml(__('Today's date: %1', $block->getCurrentDate())) ?>
</p>
```

Reply:
Thats strange I get the current date on my computer

Reply:
'cause you have the disabled `full_page` cache. check it in your terminal
```
bin/magento cache:status
```

Reply:
even if it's really working, you can't push your module changes and I can't review them. and when I'll merge your changes with a non-existing PHP class that is used for your block in XML, the deploying process will be broken. that's why I asked you not to do any BE changes that require a module, not only the theme

Reply:
Ok I think I understand. I did it because it helps me get a better understanding of Magento in general. How everything is connected and where/how all the information comes from. I now understand a little bit better how PHTML and XML files work. I enabled the cache and it still work.

Reply:
yeah, I like your motivation and really support it. I just can't merge your changes, 'cause they will break the deploying process 😀
so, please 
- remove the custom class from XML
- change the class in the PHTML template for `@var`
- and remove the used custom function to get the date from the PHTML template
then I can approve the PR and will merge your changes

Reply:
Ok nice. I'm was not sure how you mean so I just deleted the whole Code directory and the footer_links in default.xml and custom_footer.phtml. I have already added @var to every PHTML template file. :)